src/main/java/com/project/videotecha/exception/RestResponseEntityExceptionHandler.java

```diff
@@ -31,4 +31,10 @@ public ExceptionDto handleMethodValidationFailure(MethodArgumentNotValidExceptio
                 .forEach(e -> b.append(e.getDefaultMessage()).append(";"));
         return new ExceptionDto(b.toString(), Instant.now(), HttpStatus.BAD_REQUEST.value(), HttpStatus.BAD_REQUEST);
     }
+
+    @ExceptionHandler(value = EmailNotSentException.class)
+    public ExceptionDto handleEmailNotSentException(EmailNotSentException e) {
+        return new ExceptionDto(e.getMessage(), Instant.now(), HttpStatus.UNPROCESSABLE_ENTITY.value(), HttpStatus.UNPROCESSABLE_ENTITY);
```

Comment:
Why `HttpStatus.UNPROCESSABLE_ENTITY`?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/422

Since we have email validation present the thought was that the only error that could occur would be some kind of error on the side of the service used to send emails(i.e. gmail's smtp server). 

This status code would then indicate that the error wasn't with the information provided, but with the external server.